app/src/initializeApp.js

```diff
@@ -37,7 +37,16 @@ function setOptions(urlParams, store) {
   }
 }
 
+export const bootError = (error) => {
+  document.querySelector("#app").innerText = `
+    Lunie failed to start. Please contact us at http://help.lunie.io/
```

Comment:
```suggestion
    Uh oh. Lunie failed to start. Our team has been notified of the issue. Please try refreshing this page in a few minutes. 
```

what about something like this?

Reply:
You don't want to give a way to contact us?